test/functional/tests/io_class/test_io_class_preserve_config.py

```diff
@@ -1,6 +1,6 @@
 #
-# Copyright(c) 2022 Intel Corporation
-# SPDX-License-Identifier: BSD-3-Clause-Clear
+# Copyright(c) 2022-2022 Intel Corporation
```

Comment:
No need to add date range if it's the same year.

Reply:
Not ranged year was failing the license checker. I guess it is because this file was edited on two different dates in the same year.

Reply:
License checker was failing because there was an old license (BSD-3-Clause-Clear). If copyright checker was also failing on single year it means that copyright checker needs to be fixed. Can you test it once again?

Reply:
Tested again with single year, it passes.